sdks/java/build-tools/src/main/resources/beam/checkstyle.xml

```diff
@@ -59,6 +59,9 @@ page at http://checkstyle.sourceforge.net/config.html -->
     <property name="onCommentFormat" value="CHECKSTYLE.ON\: ([\w\|]+)"/>
     <property name="checkFormat" value="$1"/>
   </module>
+  <module name="SuppressionFilter">
```

Comment:
Does this supersede the pom.xml `suppressionsLocation`? Please remove the old config if no longer needed.